.gitignore

```diff
@@ -298,10 +298,13 @@ ASALocalRun/
 .mfractor/
 
 # CMake
-build/
+/build/
 /cmake-build-debug
 
-#Doxygen
+# Visual Studio build output
+/out/
```

Comment:
we probably don't need this.
Form line 1 to line 100, all of the ignore configuration is to make sure that anything that visual studio can generate is ignored.

Customer can change the name of the default folder, (from 'out' to 'myBuild' for example) and this becomes useless.
Instead, we describe the entire list of files to be ignored from VS

Reply:
> Form line 1 to line 100, all of the ignore configuration is to make sure that anything that visual studio can generate is ignored.

`/out/` isn't specified there. I will move it up to that section.

> Customer can change the name of the default folder, (from 'out' to 'myBuild' for example) and this becomes useless.
Instead, we describe the entire list of files to be ignored from VS

Since that is the default, I think we should ignore it. The only reason it is currently getting ignored is because we **also** ignore the `/build`/ directory. If we remove `/build/` from the .gitignore, the `/out/` folder starts getting tracked.